science/cdo/Portfile

```diff
@@ -28,12 +31,14 @@ long_description \
 
 fetch.ignore_sslcert        yes
 
-# Since cdo 2.4.0, we need to select C++20 capable compilers.
-# On MacOS-13 compilation with /usr/bin/clang++ fails, so we blacklist it.
-# https://github.com/macports/macports-ports/pull/24069
-
+# Since version 2.4.0, cdo requires C++20 capable compilers.
+# https://code.mpimet.mpg.de/news/536
 compiler.cxx_standard       2020
-compiler.blacklist-append   clang
+
+# With that in place, on MacOS-13, compilation with /usr/bin/clang++ still failed
+# so we also blacklist all clang < 1500.
+# https://github.com/macports/macports-ports/pull/24069
+compiler.blacklist-append   { clang < 1500 }
```

Comment:
This `compiler.blacklist-append` line does nothing because when `compiler.cxx_standard` is set to `2020`, MacPorts will not use any clang < 1500. See https://trac.macports.org/wiki/CompilerSelection#LanguageStandards